README.md

```diff
@@ -1,3 +1,8 @@
+<p align="center"><img src="https://github.com/jetstack/cert-manager/blob/master/logo/logo.png" width="250x" /></p>
+</a>
+<a href="https://godoc.org/github.com/cert-manager/csi"><img src="https://godoc.org/github.com/cert-manager/csi?status.svg"></a>
```

Comment:
I think these are pointing to the wrong repo. 

I also believe godoc.org is going away at some point in future?

Reply:
Thanks, I wasn't aware it was going away...